superbench/benchmarks/model_benchmarks/pytorch_base.py

```diff
@@ -35,10 +44,256 @@ def __init__(self, name, parameters=''):
         self._framework = Framework.PYTORCH
         torch.backends.cudnn.benchmark = True
 
+        self._generate_log = False
+        self._compare_log = None
+        self._model_run_metadata = {}
+        self._model_run_losses = []
+        self._model_run_periodic = {}
+
     def _judge_gpu_availability(self):
         """Judge GPUs' availability according to arguments and running environment."""
         self._gpu_available = not self._args.no_gpu and torch.cuda.is_available()
 
+    def _enable_deterministic_training(self):
+        """Enable deterministic training settings for reproducible results."""
+        if hasattr(self._args, 'deterministic_seed'):
+            torch.manual_seed(self._args.deterministic_seed)
+            random.seed(self._args.deterministic_seed)
+            if torch.cuda.is_available():
+                torch.cuda.manual_seed(self._args.deterministic_seed)
+                torch.cuda.manual_seed_all(self._args.deterministic_seed)
```

Comment:
Looks ` torch.cuda.manual_seed_all(self._args.deterministic_seed)` is enough here. Can we remove `torch.cuda.manual_seed(self._args.deterministic_seed)`?